client/src/Movies/Movie.js

```diff
@@ -11,6 +13,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
+    this.fetchMovie(this.props.match.params.id)
     const id = 1;
```

Comment:
So you need to remove everything after line sixteen now. Line sixteen is going to do what you want and fetching the movie, but then what are line seventeen and eighteen doing? So you should just have line sixteen, or do:
`const { id } = this.props.match.params;`
`this.fetchMovie(id);`

Make sense?